eodhp-web-presence/templates/web-presence-deploy.yaml

```diff
@@ -63,3 +63,9 @@ spec:
               value: {{ .Values.hosting.allowed_hosts | default "localhost,127.0.0.1" }}
             - name: BASE_URL
               value: {{ .Values.hosting.base_url | default "http://127.0.0.1" }}
+            - name: PAGE_CACHE_LENGTH
+              value: {{ .Values.caching.page_length | default "300" }}
+            - name: STATIC_FILE_CACHE_LENGTH
+              value: {{ .Values.caching.static_file_length | default "3600" }}
+            - name: IS_PROD
+              value: {{ .Values.is_prod | default "0" }}
```

Comment:
`{{ .Values.is_prod | default 0 }}`  as with the quotation marks it would be a string and resolved as `True`.

Reply:
Updated